src/Fixer/ClassNotation/ClassDefinitionFixer.php

```diff
@@ -294,14 +296,21 @@ private function fixClassyDefinitionOpenSpacing(Tokens $tokens, array $classDefI
      *     extends: false|array{start: int, numberOfExtends: int, multiLine: bool},
      *     implements: false|array{start: int, numberOfImplements: int, multiLine: bool},
      *     anonymousClass: bool,
+     *     final: false|int,
+     *     abstract: false|int,
+     *     readonly: false|int,
      * }
      */
     private function getClassyDefinitionInfo(Tokens $tokens, int $classyIndex): array
     {
+        $tokensAnalyzer = new TokensAnalyzer($tokens);
         $openIndex = $tokens->getNextTokenOfKind($classyIndex, ['{']);
         $extends = false;
         $implements = false;
         $anonymousClass = false;
+        $final = false;
+        $abstract = false;
+        $readonly = false;
```

Comment:
```suggestion
```
This could be removed entirely and `$final ?? false` could be used in the `return`ed array.

Reply:
I don't see how, I tried and the tests no longer pasted.

Reply:
Weird, because using fallback value for undefined variables should work the same as defining initial value for them 🤔 Will try it later.